config/environments/demo.rb

```diff
@@ -1,6 +1,9 @@
 require_relative "./shared_deployment_config"
 
 Rails.application.configure do
+  config.public_file_server.headers = {
+      'Cache-Control' => 'public, max-age=31536000'
+  }
```

Comment:
If this belongs in both demo and prod, it might make more sense to put it in `shared_deployment_config.rb`

Reply:
ah my bad for some reason i thought that included staging but i see that's separate. i'll do that.